src/core/loader/smdh.cpp

```diff
@@ -53,6 +53,11 @@ SMDH::GameRegion SMDH::GetRegion() const {
         return GameRegion::RegionFree;
     }
 
+    constexpr u32 taiwan_and_china = (1 << GameRegion::Taiwan) & (1 << GameRegion::China);
```

Comment:
This should build and has proper formatting:
```suggestion
    constexpr u32 taiwan_and_china =
        (1 << static_cast<u32>(GameRegion::Taiwan)) & (1 << static_cast<u32>(GameRegion::China));
```